pulseapi/creators/migrations/0002_auto_20161219_2146.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
```

Comment:
as a new model, these two migrations can be collapsed (delete 001 and 002, then rerun makemigrations)